README.md

```diff
@@ -147,6 +147,7 @@ bash
 
 - [ ] *New content*
 - [ ] *Testimonial section*
+- [ ] *likes icon to check which continent is most visited*
```

Comment:
- [ ] You did a great job on this milestone but there are some things you still need to address 
Kindly The video contains a highlight about a piece of the code the student finds interesting or is proud of making please make a clear explanation about a pieces of the code you find interesting or proud of explanation with a cogest point of the lines of code .

Reply:
Good evening dear @ademibowale,
I hope this message finds you well,
can you please confirm me if I do need to make the video again showing the pieces of code 
i find interesting ?
Thank you in advance,

Reply:
@IvonneBenitesRodriguez you did well on your presentation but you only need to make a bit more  explanation of your piece of the code you love ,so just make new video ,this presentation video will be useful for you later please give proper explanation of the pieces of the code you love .

Reply:
Dear @ademibowale,
Thank you so much for replying, I truly appreciate it, 
I have made a new video with the suggestion you mentioned, 
thank you again, 
Best wishes,